### ISSUE TITLE
Add Dotnet Templates with EFCore

### ISSUE THREAD

**Issue by jkone27** [efcore/EFCore.FSharp#99]
does this have to rely on Paket, is it available as a dotnet template for mvc? that would be great

```
dotnet new efcore-mvc --language f# 
```

and also

```
dotnet new efcore-saturn --language f# 
```

but I think paket could be kept out from simple templates to make it more accessible to "standard" .net devs.
In the end EF is the battle-proof db access .NET orm standard...  so it would be handy to have it in F# as a "default" choice

**Comment by bricelam** [comment]
Oh I like the idea of adding it to the [StarterWeb-FSharp template](https://github.com/dotnet/aspnetcore/tree/main/src/ProjectTemplates/Web.ProjectTemplates) behind the `--auth Individual` option like it is in C#. Even if the future of that option is [a little uncertain](https://devblogs.microsoft.com/aspnet/asp-net-core-6-and-authentication-servers/)...

**Comment by simon-reynolds** [comment]
Definitely! I'll see if I can put together a PR to add support for that
